test/layouts/test_slice.py

```diff
@@ -150,3 +150,15 @@ def test_command_propagation(manager):
     org_height = manager.c.window.info()["height"]
     manager.c.layout.toggle_split()
     assert manager.c.window.info()["height"] != org_height
+
+
+@slice_config
+def test_command_propagation_direct_call(manager):
+    manager.test_window("slice")
+    manager.test_window("one")
+    manager.test_window("two")
+    info = manager.c.layout.info()
+    assert info["name"] == "slice"
+    org_height = manager.c.window.info()["height"]
+    manager.c.layout.eval("self.toggle_split()")
```

Comment:
`toggle_split` is exposed as a command by `Stack`. Should we also test with `manager.c.layout.toggle_split()`?

Reply:
I'm confused. Why is this a question for `test_slice`?

Reply:
Because line 163 is calling `Slice.toggle_split()`, which falls through `Slice.__getattr__` to `Stack.toggle_split()`

Reply:
Ah. OK. We do actually test `manager.c.layout.toggle_split()` in line 151 (we have a test which checks if it works via the command interface and another to check if it works with a direct call).